tests/test_item_resource.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+import requests
+import re
+
+from urllib.parse import urljoin
+
+
+class TestItemResourceJson(object):
+    @pytest.fixture
+    def response(self, endpoint):
+        self.register_name = re.sub(r'http[s]?://([^\.]+)(.*)', r'\1', endpoint)
+
+        entry = requests.get(urljoin(endpoint, 'entry/1.json'))
```

Comment:
I was wondering if we might be able to make this request a shared fixture used by both entry and item tests. I'm not sure if there's a good way of doing it though.